generators/common/files.js

```diff
@@ -1,81 +0,0 @@
-/**
- * Copyright 2020-2023 the original author or authors from the JHipster project.
- *
- * This file is part of the JHipster project, see https://www.jhipster.tech/
- * for more information.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-const { pathFromSvelteBlueprint } = require('../util');
-
-const prettierConfigFiles = {
-	global: [
-		{
-			templates: ['.prettierrc', '.prettierignore'],
-		},
-	],
-};
-
-const commonFiles = {
-	global: [
-		{
-			templates: ['.editorconfig', '.gitignore', 'README.md', 'sonar-project.properties'],
```

Comment:
file override are looked at blueprints by the main generator.
Many files uses partial files at the specific generator, svelte in this case.

Reply:
I noticed the use of partials in `quarkus` blueprint. It may work for files/scenarios that require customization, however, it may not be best suited when you need to override the complete file. `JHipster` generates too much verbose documentation at all levels and that's something I plan to reduce in this blueprint. For now, lets see how the application is generated and will comment if I observe any difference wrt to existing applications